internal/userutils/userutils.go

```diff
@@ -0,0 +1,40 @@
+// Package userutils provides functions related to system users and groups.
+package userutils
+
+import (
+	"errors"
+	"fmt"
+	"os"
+)
+
+// GroupFile is the path to the group file. It is exported for testing purposes.
+var GroupFile = "/etc/group"
```

Comment:
just keep it unexported and add an `export_test.go` file with it.

Reply:
Actually... No, keep it exported but change the comment and use it also in 

```diff
diff --git a/internal/users/localentries/localgroups.go b/internal/users/localentries/localgroups.go
index 8c3b5b86..1740914a 100644
--- a/internal/users/localentries/localgroups.go
+++ b/internal/users/localentries/localgroups.go
@@ -13,12 +13,13 @@ import (
 	"sync"
 
 	"github.com/ubuntu/authd/internal/sliceutils"
+	"github.com/ubuntu/authd/internal/userutils"
 	"github.com/ubuntu/authd/log"
 	"github.com/ubuntu/decorate"
 )
 
 var defaultOptions = options{
-	groupPath:    "/etc/group",
+	groupPath:    userutils.GroupFile,
 	gpasswdCmd:   []string{"gpasswd"},
 	getUsersFunc: getPasswdUsernames,
 }
```

Reply:
While `internal/users/db/migration.go` should probably not have to import it instead but `renameUsersInGroupFile` should return an error mentioning it in case, no?